src/components/augmentedReality/AugmentedRealityView.js

```diff
@@ -125,40 +125,45 @@ const ViroSoundAnd3DObject = (item) => {
             <ViroVideo
               loop
               materials={['chromaKeyFilteredVideo']}
-              position={object?.mp4?.position}
-              rotation={object?.mp4?.rotation}
-              scale={object?.mp4?.scale}
-              source={{ uri: object?.mp4?.uri }}
+              position={object.mp4?.position}
+              rotation={object.mp4?.rotation}
+              scale={object.mp4?.scale}
+              source={{ uri: object.mp4?.uri }}
             />
           )}
 
           {!!object?.image && (
             <ViroImage
-              position={object?.image?.position}
-              rotation={object?.image?.rotation}
-              scale={object?.image?.scale}
-              source={{ uri: object?.image?.uri }}
+              position={object.image?.position}
+              rotation={object.image?.rotation}
+              scale={object.image?.scale}
+              source={{ uri: object.image?.uri }}
             />
           )}
         </>
       )}
 
-      <Viro3DObject
-        source={{ uri: object?.vrx?.uri }}
-        resources={object?.textures}
-        type="VRX"
-        position={object?.vrx?.position}
-        rotation={object?.vrx?.rotation}
-        scale={object?.vrx?.scale}
-        onLoadStart={() => setIsObjectLoading(true)}
-        onLoadEnd={() => setIsObjectLoading(false)}
-        onError={() => alert(texts.augmentedReality.arShowScreen.objectLoadErrorAlert)}
-        animation={{
-          loop: true,
-          name: object?.animationName,
-          run: isStartAnimationAndSound
-        }}
-      />
+      {!!object?.models &&
+        !!object?.textures &&
+        object.models.map((model, index) => (
+          <Viro3DObject
+            key={index}
+            source={{ uri: model.uri }}
+            resources={object.textures}
+            type="VRX"
+            position={model.position}
+            rotation={model.rotation}
+            scale={model.scale}
+            onLoadStart={() => setIsObjectLoading(true)}
+            onLoadEnd={() => setIsObjectLoading(false)}
+            onError={() => alert(texts.augmentedReality.arShowScreen.objectLoadErrorAlert)}
+            animation={{
+              loop: true,
+              name: object?.animationName,
```

Comment:
animation name stays directly at `object`?

Reply:
Yes, it is directly inside the object. But in the last update, when updating `localUris`, we forgot to update here. So I'm updating it immediately.

Actually `animationName` information is in the `payload`. Right now we want `localUris.animationName`. But `localUris` is an array and there is no `animationName` information in it. Now I wrote `payload` instead of `localUris` and the problem is fixed.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/706/commits/958199c5725ff2f7d22e0199b00113d37c3c744e